cvat-ui/src/components/annotation-page/attribute-annotation-workspace/attribute-annotation-sidebar/attribute-annotation-sidebar.tsx

```diff
@@ -205,7 +205,7 @@ function AttributeAnnotationSidebar(props: StateToProps & DispatchToProps): JSX.
 
     const siderProps: SiderProps = {
         className: 'attribute-annotation-sidebar',
-        theme: 'light',
+        theme: 'dark',
```

Comment:
Is there a darker theme? :P

Reply:
Nope this is the darkest theme that they got in China currently :P